docs/reference/indices/recovery.asciidoc

```diff
@@ -90,7 +90,8 @@ Response:
         "repository" : "my_repository",
         "snapshot" : "my_snapshot",
         "index" : "index1",
-        "version" : "{version}"
+        "version" : "{version}",
+        "restoreUUID": ""
```

Comment:
I added this field to the xcontent serialization because we serialize all fields in the data structures in general, not sure if it makes sense to expose it here though tbh.